gator/invoke.py

```diff
@@ -285,6 +285,80 @@ def invoke_all_fragment_checks(
     return met_or_exceeded_count
 
 
+def invoke_all_regex_checks(
+    fragment,
+    expected_count,
+    filecheck=NOTHING,
+    directory=NOTHING,
+    contents=NOTHING,
+    exact=False,
+):
+    """Perform the check for a regex existence in file or contents and return the results"""
+    met_or_exceeded_count = 0
+    met_or_exceeded_count, actual_count = fragments.specified_fragment_greater_than_count(
```

Comment:
If you're reusing the fragment method, it should probably be refactored (since fragment is a specialized version of the regex check). It's name should be something like `specified_entity_greater_than_count`.